docs/adrs/0078-develop-strategy-for-ensuring-code-test-coverage.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 0078 Develop strategy for ensuring code test coverage
+description: |
+  Decision outcome: The Happo Approach
+---
+
+# Develop strategy for ensuring code test coverage
+
+**User Story:** *[MB-16176](https://dp3.atlassian.net/browse/MB-16176)* :lock:
+
+## Problem Statement
+
+Our current system of maintaining test coverage involves measuring the current total test coverage and ensuring that any given PR does not decrease the total coverage.
+This strategy has obvious benefits in helping to support a high test coverage, but it can also introduce unnecessary roadblocks in the PR workflow. To understand this, it's helpful to understand the scenarios that might result in a decrease in total coverage.
```

Comment:
Re-wording the problem statement a bit here to better represent the issue we've faced.

```suggestion
This strategy has obvious benefits in preventing test coverage from going down, but it has also introduced roadblocks in the PR workflow. To understand this, it's helpful to understand the scenarios that might result in a decrease in total coverage.
```